docs/admin/users/github-auth.md

```diff
@@ -87,7 +99,7 @@ Alternatively, if you are running Coder as a system service, you can achieve the
 same result as the command above by adding the following environment variables
 to the `/etc/coder.d/coder.env` file:
 
-```env
```

Comment:
this should stay env

Reply:
different doc, but here's what it looks like with `env` on the left, and on the right is `shell`:

![2025-05-29_09-26-01](https://github.com/user-attachments/assets/30c5f13a-37a5-4b32-9afb-64a2075b0587)


here's a list of the supported languages https://prismjs.com/#supported-languages